packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -1,4 +1,78 @@
-{{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  {{!-- {{yield}} --}}
+
+  {{! Hds-TabList }}
+  <div class="hds-tab-list">
```

Comment:
[question] What does this wrapper do?

Reply:
It does nothing currently but eventually I'll need to add overflow navigation so it would be a wrapper to contain that navigation together with the tabs. Some parts of the DOM may get slightly refactored as I work to implement all the various parts though...